geometry_msgs/Pose robot_pose
float32 PTU_pan
float32 PTU_tilt
